docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+   cinema:
+       build:
+           context: .
+       ports:
+           - "8000:8000"
+       command: >
+           sh -c "python manage.py wait_for_db &&
+                  python manage.py migrate &&
+                  python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
+
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+
+   db:
+       image: postgres:16.0-alpine3.17
+       restart: always
+       ports:
+           - "5433:5432"
```

Comment:
The database port is mapped to `5433` on the host. Ensure this does not conflict with other services and is correctly configured in your environment variables.